README.md

```diff
@@ -1,10 +1,40 @@
-# fortress
+# Fortress
 
-This repo contains code for authenticating with [Hashicorp Vault](https://www.vaultproject.io/) and fetching secrets. The code can be run as a script at commandline, and it can also be used by Python applications as a package.
+This repo contains code for authenticating with [Hashicorp Vault](https://www.vaultproject.io/) and fetching secrets. The code can be run as a script at the command-line, and it can also be used by Python applications as a package.
 
 The script is run in a Kubernetes Init Container for [Artsy apps](https://github.com/artsy/hokusai-sandbox/blob/e04dc52c7e5cd0144c34ba0541e7d32fc84bc15f/hokusai/staging.yml#L35).
 
 * __Point People:__ [#product-sapphire][sapphire_channel]
 
 
+## Setup
+
+Read and run the setup script:
+```
+./bin/setup.sh
+```
+
+## Run Script
+
+To run the script with environment files loaded (using [Foreman](https://github.com/ddollar/foreman)):
+
+```
+foreman run --env .env.shared,.env python src/load/load.py iam staging hokusai-sandbox
```

Comment:
We tend to use [python-dotenv](https://github.com/theskumar/python-dotenv), as it's a python dependency which can be specified as a dev dependency by the project.

Reply:
Right... in prod we do not use dotenv files in data platform, we use either configmaps or in the case of AWS Batch, we have job configurations. You can then just use an env var simply named `ENV` to distinguish which environment it's running in, which defaults to `development` of course.

Reply:
[updated]([dd9fb64](https://github.com/artsy/fortress/pull/3/commits/dd9fb64d33bc9cb684129e35e8e8afcc95f013aa)). opted default to None since in staging/prod code doesn't behave specifically.